tests/js/spec/components/sidebar/__snapshots__/index.spec.jsx.snap

```diff
@@ -6517,11 +6517,11 @@ exports[`Sidebar can have onboarding feature 1`] = `
                                                     onMouseLeave={[Function]}
                                                   >
                                                     <ForwardRef
-                                                      color="#FA5849"
+                                                      color="redLight"
                                                       size="xs"
                                                     >
                                                       <ForwardRef(SvgIcon)
-                                                        color="#FA5849"
+                                                        color="redLight"
```

Comment:
Is this right?

Reply:
i think so? shallow snapshot will now show the name of the color being used

Reply:
it's correct 😉